GPU/Common/TextureCacheCommon.cpp

```diff
@@ -804,7 +804,11 @@ void TextureCacheCommon::NotifyFramebuffer(u32 address, VirtualFramebuffer *fram
 					depth ? "DEPTH" : "COLOR", (int)candidates.size(), addr, framebuffer->width, framebuffer->height, depth ? framebuffer->z_stride : framebuffer->fb_stride, GeBufferFormatToString(framebuffer->format));
 			}
 
-			AttachBestCandidate(candidates);
+			// There can actually be multiple ones to update here! This can be the case where two textures point to different framebuffers that share depth buffers.
+			// So we have no choice but to run all the candidate matches.
```

Comment:
I guess the reporting isn't useful then...

-[Unknown]

Reply:
I suppose we could detect this particular case, and report if it's not that...